src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGenerator.java

```diff
@@ -149,6 +164,28 @@ public DirectionsQuestionnaireForm getTemplateData(CaseData caseData) {
             .build();
     }
 
+    private boolean isClaimantResponse(CaseData caseData) {
+        if (caseData.getApplicantsProceedIntention() != null) {
```

Comment:
maybe refactor to 
return caseData.getApplicantsProceedIntention() != null

Reply:
Done